packages/provider-utils/src/is-abort-error.ts

```diff
@@ -1,6 +1,6 @@
 export function isAbortError(error: unknown): error is Error {
   return (
-    error instanceof Error &&
+    (error instanceof Error || error instanceof DOMException) &&
```

Comment:
Please let me know if I'm missing context, since the AbortError error created in the structured-object.svelte instantiates a DOMException not Error, so `isAbortError` would return false.

Happy to change this if there's a better way

Reply:
are there envs where DOMException is not available and this could cause errors?